dependency-graph/properties/dependency-review.properties.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "Dependency Review",
+    "creator": "GitHub",
+    "description": "Scans Pull Requests on each push for the introduction and/or resolution of vulnerable depdendencies to the repository",
+    "iconName": "octicon mark-github",
+    "categories": [
+      "Dependency Graph",
+      "Dependency Review",
+      "Go",
+      "Java",
+      "JavaScript",
+      "TypeScript",
+      "Python",
+      "Ruby",
+      "Actions",
+      "PHP"]
```

Comment:
I assume you have covered all the languages which are supported by DR action?

Reply:
That said, I'll take another pass to see if there are more appropriate tags we should include anyway thanks for the callout 🙇